src/rgw/rgw_sync.cc

```diff
@@ -1393,6 +1393,8 @@ class RGWCloneMetaLogCoroutine : public RGWCoroutine {
   int state_store_mdlog_entries_complete();
 };
 
+#define META_SYNC_SPAWN_WINDOW 20
```

Comment:
is this the new bound?

Reply:
yes, this is the max number of metadata entries we'll sync at a time, per mdlog shard